helium wallet migration guide

samgutentag opened the pull request.

samgutentag committed:
Solana migration guides for Helium Wallets

samgutentag committed:
added known maker app links

samgutentag committed:
moved context section

cokes518 commented on docs/wallets/wallets.mdx:
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
-In blockchain projects, various wallet applications exist to help users manage their own private
-keys, enabling them to sign transactions and transfer assets.
+All Hotspots on the Helium Network with an animal name will automatically become a
```
this feels like a very abrupt introduction to "Helium is moving to Solana". Can there be a little more preamble?

samgutentag replied:
preamble added, may need some more work. but also didn't want to deviate too far off topic from Wallet stuff to general L1 migration stuff

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
### Hotspots Will Earn IOT
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
- Users can no longer onboard new Hotspots using the Helium App. All new Hotspots will be directed to use the Hotspot's Maker app, web dashboard, or contact email support (in the event that a Maker app is not available). 

:::note
All Makers had a deadline to launch their Maker app by March of 2022. Makers without an app or web dashboard either have an app in development or have not responded to requests for additional updates. 
:::


```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
Seed phrase export functionality will remain in the app for at least 1 year post-migration.
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
2. Tap `Start Migration` in the migration popup
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
3. Confirm the Wallet account you want to import, then tap `Export to Wallet App`. The [Helium Wallet app](/wallets/helium-wallet-app) will automatically open and complete the migration.
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
4. Now your HNT, DC, MOBILE, and Hotspots have successfully moved to the Helium Wallet App and ready for the Solana migration, which will happen automatically in January 2023.
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
Hotspot NFTs in the NFT Collectibles section once Helium on Solana goes live. Token balances are visible on the account page.
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
You will use the [Helium Wallet app](/wallets/helium-wallet-app) moving forward. You can find
```

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
Your public wallet address will change post-migration, but your private key and 12-word seed phrase will remain the
```

samgutentag committed:
Apply suggestions from code review

Co-authored-by: Coco Tang <coco@helium.com>

samgutentag committed:
content and formatting

- specify blockchain vs Wallet migration for clarity
- Wallet is a proper noun
- additional preamble for Solana blockchain migration
- admonition formatting and titles
- link to Solana docs for CLI Wallet users
- line wrap and formatting

samgutentag committed:
added app store and play store link badges

cokes518 commented on docs/wallets/helium-wallet-app.mdx:
```diff
@@ -10,9 +10,43 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
+
+:::
+
 The Helium Wallet app is the new trusted way to manage your tokens and identity on the Helium
```
```suggestion
The Helium Wallet app is the trusted way to manage your tokens and identity on the Helium
```

cokes518 commented on docs/wallets/helium-wallet-app.mdx:
```diff
@@ -10,9 +10,43 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
+:::
+
 The Helium Wallet app is the new trusted way to manage your tokens and identity on the Helium
 Blockchain. This new Wallet App is not meant to manage Hotspots or Validators.
```
```suggestion
Blockchain. 
```

cokes518 commented on docs/wallets/wallets.mdx:
screenshots sent offline.

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
Your public key will change on Solana but the private key will remain the same.
```

samgutentag committed:
Apply suggestions from code review

Co-authored-by: Coco Tang <coco@helium.com>

samgutentag committed:
content and image updates

- spacer bars to visually break up sections
- image samples of Helium app migration flow
- removed over zealous cross links
- app mockup images
- reorg image files, optimzed file sizes

cokes518 commented on docs/wallets/wallets.mdx:
```suggestion
1. Export Seed Phrase or private key to import into a Solana CLI Wallet (or the Helium Wallet app)
```

cokes518 committed:
Update docs/wallets/wallets.mdx
